winum.el

```diff
@@ -276,6 +280,7 @@ There are several ways to provide the number:
 - if called interactively and no valid argument is provided, read from
   minibuffer."
   (interactive "P")
+  (winum--check-for-scope-change)
```

Comment:
I don't think this one is needed, `winum-get-number` and `winum-get-window-by-number` are both called unconditionally right after.